source/process/feature-release.md

```diff
@@ -21,49 +21,54 @@ Pre-work for the current release begins at the code complete date of the previou
 
 ### B. (T-minus 18 working days) Cut-off for submitting features
 
-No pull requests for major features should be **submitted** to the current release after this date. In special cases, exceptions can be made by the Release Manager. 
+No pull requests for major features should be **submitted** to the current release after this date. In special cases, exceptions can be made by the QA Release Manager. 
 
-1. Release Manager:
+1. QA Release Manager:
     - Post this checklist in Release Checklist channel
+2. QA - For each team:
     - Follow that major feature PR reviews are prioritized and post a list of outstanding feature PRs in the Release Discussion channel
```

Comment:
Maybe this and the next one can be done by Teams (since they know what they're working on / what's on their sprint)?

Reply:
Updated, let me know what you think